core/src/main/java/nl/nn/adapterframework/core/Adapter.java

```diff
@@ -240,18 +240,18 @@ public boolean configurationSucceeded() {
 	}
 
 	/**
-	 * sends a warning to the log and to the messagekeeper of the adapter
+	 * send a warning to the log and to the messagekeeper of the adapter
 	 */
 	protected void warn(String msg) {
 		log.warn("Adapter [{}] {}", name, msg);
 		getMessageKeeper().warn(msg);
 	}
 
 	/**
-	 * sends a warning to the log and to the messagekeeper of the adapter
+	 * send an error to the log and to the messagekeeper of the adapter
 	 */
 	protected void addErrorMessageToMessageKeeper(String msg, Throwable t) {
```

Comment:
Deze methode wordt op andere plekken ook aangeroepen, nu laat hij nooit meer de stacktrace zien. Is dat wel verstandig?

Reply:
Ik dacht dat je deze aanraadde om aan te passen, in je chat op Teams. Maar idd wordt deze vaker gebruikt en dan zonder dat anderen de stacktrace loggen. Ik zal deze terugdraaien.

Reply:
Ik stel voor
`addErrorMessageToMessageKeeper("error processing message with messageId [" + messageId+"]: ",e);`
te vervangen door
`warn("error processing message with messageId [" + messageId+"]: " + e.getMessage());`